versions.gradle

```diff
@@ -26,13 +26,13 @@ def versions = [:]
 // https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/
 versions.gecko_view_nightly = "115.0.20230605094751"
 versions.gecko_view_beta = "114.0.20230525180143"
-versions.android_components = "110.1.0"
+versions.android_components = "116.0b2"
```

Comment:
I think we can squash this commit with the first one. Looks weird to update the version twice in a single PR (although the first commit is already too large)